### DISCUSSION TITLE
Feature request (nix): Make different builds available in flake

### DISCUSSION THREAD

**Discussion by FantomeBeignet** [ghostty-org/ghostty#4073] [Feature Requests, Ideas] [answered]
Would it be possible to make different builds (ie. with or without libadwaita/X11 support) available in the Nix flake ? Some people don't need the adwaita features or the X11 support, so having the ability to avoid building ghostty with them will only make the install/update process faster/smoother for them.

**Comment by rhodes-b** [comment]
The flake is use for the CI build environment for ghostty so I view this as very unlikely. This is something that would be added to the package in nixpkgs, which I believe people have been talking about

**Comment by jcollie** [comment, marked as answer]
The "official" package in Nixpkgs allows for much more customization.

**Reply by FantomeBeignet** [reply]
Alright, I'll keep using the flake for now as development is still going fast and there're a few things I'm waiting for, but I'll switch to the "official" package in the future.
